doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -403,60 +417,209 @@ Response code:
 
 + Parameters
-    + entityId: Bcn_Welt (required, string) - Entity Id.
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + op: append (optional, string) - Strict append.
 
-+ Response 404 (application/json)
++ Request (application/json)
+
+    + Body
+
+            {
+                "ambientNoise": 31.5
+            }
+
++ Response 204
+
+### Update existing entity attributes [PATCH /v2/entities/{entityId}]
+
+The request payload is an object representing the attributes to update. The object follows
+the JSON entity representation format (described in a section above), except that `id` and `type`
+are not allowed.
+
+The entity attributes are updated with the ones in the payload. In particular, the entity attributes
+are updated with the ones in the payload.In addition to that, in the case some of the attributes in the
+payload doesn't exist in the entity, an error if returned.
+
+Response:
+
+* Successful operation uses 204 No Content
+* Error use a non-200 and (optionally) an error payload. See subsection on Error Responses for more detail.
+
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+
++ Request (application/json)
 
         {
-            "error": "NotFound",
-            "description": "There is no entity with such an id"
+            "temperature": 25.5,
+            "seatsNumber": 6
         }
 
-### Retrieve entity data by id and type [GET /v2/entities/type/{entityType}/id/{entityId}]
++ Response 204
+
+### Replace entity attributes [PUT /v2/entities/{entityId}]
+
+The request payload is an object representing the attributes to replace. The object follows
+the JSON entity representation format (described in a section above), except that `id` and `type`
+are not allowed.
+
+The attributes previously existing in the entity are removed and replaced by the ones in the
+request.
+
+Response:
+
+* Successful operation uses 204 No Content
+* Error use a non-200 and (optionally) an error payload. See subsection on Error Responses for more detail.
 
 + Parameters
-    + entityId: Boe_Idearium (required, string) - Entity ID
-    + entityType: Room (required, string) - Entity Type
+    + entityId: Bcn_Welt (required, string) - Entity ID
 
-+ Response 200 (application/json)
++ Request (application/json)
 
         {
-            "type": "Room",
-            "id": "Boe_Idearium",
-            "temperature": 22.5
+            "temperature": 25.5,
+            "seatsNumber": 6
         }
 
-### Get entity attributes [GET /v2/entities/{entityId}{?attrs}]
++ Response 204
+
+### Remove entity atributes [DELETE /v2/entities/{entityId}?{attrs}]
+
+<table style="width: 100%; margin: 12px 0 0 0;">
+<tr><td colspan="2"><strong>Required Parameters</strong></td></tr>
+<tr><td colspan="2">none</td></tr>
+<tr><td colspan="2"><strong>Optional Parameters</strong></td></tr>
+<tr>
+  <td style="padding-right: 40px; width: 140px;">attrs</td>
```

Comment:
Maybe this operation is a bit dangerous? E.g. a client that is used to delete attributes using it but that in some moments forget to include `?attrs=myAttr` and deletes the entity wrontly...